constance/management/commands/constance.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django import VERSION
+from django.core.exceptions import ValidationError
+from django.core.management import BaseCommand, CommandError
+from django.utils.translation import ugettext as _
+
+from ...utils import get_constance_values
+from ...forms import set_constance_value
+from ... import LazyConfig
+
+
+config = LazyConfig()
+
+
+class Command(BaseCommand):
+    help = _('Get/Set In-database config settings handled by Constance')
+
+    def add_arguments(self, parser):
+        group = parser.add_mutually_exclusive_group()
+        group.add_argument("--list", action="store_true",
+                           help=_("List all Constance settings"))
+        group.add_argument("--get", dest='setting', nargs=1)
+        group.add_argument("--set", dest='setting', nargs=2, metavar=('SETTING', 'VALUE',))
+
+    def handle(self, *args, **options):
+
+        if VERSION < (1, 8):
+            # since we assume use of argparse
+            raise CommandError(_("Constance CLI requires Django 1.8 or later"))
```

Comment:
No need to mark management command output as translatable, that's an anti-pattern since CLI output should be parsable. Django doesn't mark those either.